Tools/wpe/meson-wrapper

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
```

Comment:
Better to use `/usr/bin/env sh` since not all Linux distros have `sh` located at `/bin/sh` (i.e: NixOS).